portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,141 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query;
+import java.util.*;
+
+/** Represents a datastore of comments.
+ * @author Olga Shimanskaia
+ * @author oshimanskaia@google.com
+ * @version 2.2.0
```

Comment:
Feel free to remove `@version` and `@since`, as they are usually used in larger projects that worry about version compatibility. If you want to keep them, you can change to `1`, which is your current project version (line 48 of _pom.xml_).